### ISSUE TITLE
Feature request: History filters

### ISSUE THREAD

**Issue by racuna** [renyuneyun/Easer#226]
Not a bug.

Can you add filters on the history? I want to see only "Satisfied" conditions. Currently is somewhat hard to follow the scripts in the history and look what are the ones that are being triggered.

I know, this is not a priority nor a bug, but it would be a nice improvement to the UI and I hope is not that hard to implement.

thanks

**Comment by renyuneyun** [comment]
Hi there.
It looks useful, but I'm afraid I'm not prioritized on that...
For now, you can either "clear" the history, or use the "external logging" (in Settings), and manually filter that.

But a good news is (as also mentioned in the other comment): Pivot will have more information, so I believe it will reduce the need to check the history.
